### PR TITLE
fix: Remove all the Action Pages in the issuer

### LINKED ISSUE #347: Update Index page of the issuer and remove all the Action Pages in the demo.
- Dynamic Select 
- Remove second hop from the issuer

### PR DESCRIPTION
**Pull request by talwinder50**
Screen Shot 2020-05-13 at 3 51 14 PM
Screen Shot 2020-05-13 at 4 22 13 PM

-Updated login consent pages to use tailwind css library 

closes #347

Signed-off-by: talwinder.kaur <talwinder.kaur@securekey.com>

### CODE REVIEW

File: cmd/issuer-rest/static/index.html [hunk 1]
```diff
@@ -60,28 +60,28 @@ <h2>Choose your options</h2>
...
-                        <option value="vc-issuer-interop-key"id="vci11">interop-didkey-ed25519signature2018-ed25519</option>
+                        <option value="trustbloc-ed25519signature2018-ed25519" id="vci1" selected>TrustBloc (ed25519signature2018 and ed25519)</option>
+                        <option value="trustbloc-jsonwebsignature2020-ed25519"id="vci2">TrustBloc (jsonwebsignature2020 and ed25519)</option>
+                        <option value="trustbloc-jsonwebsignature2020-p256"id="vci3">TrustBloc (jsonwebsignature2020 and p256)</option>
+                        <option value="interop-ed25519signature2018-ed25519"id="vci4">Interop (ed25519signature2018 and ed25519)</option>
+                        <option value="interop-jsonwebsignature2020-ed25519"id="vci5">Interop (jsonwebsignature2020 and ed25519)</option>
+                        <option value="interop-jsonwebsignature2020-p256"id="vci6">Interop (jsonwebsignature2020 and p256)</option>
+                        <option value="verseone-ed25519signature2018-ed25519"id="vci7">Verseone (ed25519signature2018 and ed25519)</option>
```
Comment (troyronda) [inline cmd/issuer-rest/static/index.html:71]: typo: Veres One

Comment (talwinder50) [reply cmd/issuer-rest/static/index.html]: done

File: cmd/login-consent-server/templates/consent.html [hunk 2]
```diff
@@ -36,56 +20,44 @@
...
-            </div>
+<center>
+<div class="container my-24">
+    <div class="max-w-3xl rounded shadow-lg">
+        <img class="object-cover h-48 w-full" src="img/consent.png">
+        <div class="px-6 py-4 bg-gray-100">
+            <div class="font-bold text-xl text-green-600 mb-2">An application requests access to your data</div>
 
+            <form action="" method="POST">
+            <p class="text-gray-700 text-base">Hi {{.User}}, application <strong>{{.ClientID}}</strong> wants access resources on your behalf and to:</p>
```
Comment (troyronda) [inline cmd/login-consent-server/templates/consent.html:40]: “and to:” doesn’t really fit with the following line.

maybe drop “and to”

Comment (talwinder50) [reply cmd/login-consent-server/templates/consent.html]: changed

**Pull request merged**